config/manager/manager.yaml

```diff
@@ -70,7 +70,7 @@ spec:
         - /manager
         args:
         - --leader-elect
-        image: controller:latest
+        image: $IMG
```

Comment:
Hey, this change wouldn't require the `sed` op in the Makefile right?

Reply:
Yes, this is dependent on sed. Let me put it back and I will automate deployment with daemonset bundle work.